charts/nango/values.yaml

```diff
@@ -34,33 +34,35 @@ elasticsearch:
     elasticPassword: nango
 
 server:
-  name: server
+  name: nango-self-server
```

Comment:
I would just go for nango- since it's already kind of explicit it's self-hosted

Reply:
That doesn't work for orchestrator - the env var that we use is NANGO_ORCESTRATOR_PORT so we get the same conflict with the kubernetes generated variable. The prefix needed to be something we don't use int Nango.